libr/io/io_bank.c

```diff
@@ -0,0 +1,669 @@
+/* radare2 - LGPL - Copyright 2021 - condret */
+
+#include <r_io.h>
+#include <r_util.h>
+
+#define	OLD_SM	0
```

Comment:
is this a leftover?

Reply:
uhm, kinda. But i'm not sure if I did the new way to set submap boundaries correctly

Reply:
remove it then